pycrdt_websocket/yroom.py

```diff
@@ -77,11 +78,12 @@ def __init__(
             ydoc: An optional document for the room (a new one is created otherwise).
         """
         self.ydoc = Doc() if ydoc is None else ydoc
-        self.awareness = Awareness(self.ydoc)
         self.ready_event = Event()
         self.ready = ready
         self.ystore = ystore
         self.log = log or getLogger(__name__)
+        self.awareness = Awareness(self.ydoc)
+        self.awareness.observe(self.local_update_awareness)
```

Comment:
I'm not sure about the callback name `local_update_awareness`, maybe `send_server_awareness`?